test_util/launch.py

```diff
@@ -1,20 +1,169 @@
-"""
+"""DC/OS Launch
 
 Usage:
-dcos-launch create [--wait] [--dump-info=cluster_info.json] <config=config.yaml>
-dcos-launch wait <path=cluster_info.json>
-dcos-launch describe <path=cluster_info.json>
-dcos-launch delete <path=cluster_info.json>
+  dcos-launch create [--config-path=<path>] [--info-path=<path>]
```

Comment:
- cluster-config-yaml is not suitable as this may not necessarily launch "clusters" strictly speaking
- aws-template-yaml is definitely not suitable as this will not be limited to aws and the config.yaml is not a special aws-template config.

I could go for --config-yaml-path instead but with json being a subset of yaml, you could imagine this taking a JSON file.

Perhaps --launch-config-path could work, but it just seems redundant as its there is not another config that this accepts.

I feel this should remain config-path. I loathe unnecessarily long option names and the options description below can be used to add more explanation if thats truly necessary